src/IntegrationTests/IntegrationTests/OpenTelemetryExporterSmokeTest.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Newtonsoft.Json;
+using IntegrationTests.Fixtures;
+using System.Net.Http;
+using System.Threading;
+using Xunit;
+using Xunit.Abstractions;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace IntegrationTests
+{
+    public class OpenTelemetryExporterSmokeTest : IClassFixture<OpenTelemetryUsageApplicationFixture>
+    {
+        private readonly OpenTelemetryUsageApplicationFixture _fixture;
+
+        private const string _traceApiKey = "{YOUR_TRACE_API_KEY}";
+
+        private const string _insightQueryApiKey = "{YOUR_INSIGHT_QUERY_API_KEY}";
+
+        private const string _insightQueryApiEndpoint = "https://insights-api.newrelic.com";
+
+        private const string _accountNumber = "{YOUR_ACCOUNT_NUMBER}";
+
+        public OpenTelemetryExporterSmokeTest(OpenTelemetryUsageApplicationFixture fixture, ITestOutputHelper output)
+        {
+            _fixture = fixture;
+            _fixture.TestLogger = output;
+
+            _fixture.Exercise = () =>
+            {
+                _fixture.MakeRequestToWeatherforecastEndpoint();
+            };
+
+            _fixture.SetEnvironmentVariables(new Dictionary<string, string>()
+            {
+                {"NewRelic:ApiKey", _traceApiKey}
```

Comment:
may want to add ```{"NewRelic:TraceUrlOverride", _traceEndPointUrl}``` here and
```private const string _traceEndPointUrl = "https://trace-api.newrelic.com/trace/v1";``` above for our own testing convenience so that we can set it to the staging url.

Reply:
will do that.